internal/pkg/template/templates/environment/partials/cdn-resources.yml

```diff
@@ -78,6 +92,19 @@ CloudFrontDistribution:
             {{- else}}
             OriginProtocolPolicy: match-viewer
             {{- end}}
+      {{- if .CDNConfig.Static}}
+        - Id: !Sub 'copilot-${AppName}-${EnvironmentName}-origin-s3'
+          DomainName: {{.CDNConfig.Static.Location}}
+          OriginAccessControlId: !Ref CloudFrontOriginAccessControl
+          S3OriginConfig:
+            OriginAccessIdentity: ''
```

Comment:
Do you think it's worth commenting on why there is an empty string here? I'm fine either way.